src/utils.ts

```diff
@@ -43,3 +43,14 @@ export function isFabric(): boolean {
   // @ts-expect-error nativeFabricUIManager is not yet included in the RN types
   return !!global?.nativeFabricUIManager;
 }
+
+export function isRemoteDebuggingEnabled(): boolean {
+  return (
+    // @ts-ignore WorkerGlobalScope available only when remote debugging
+    'undefined' !== typeof WorkerGlobalScope &&
+    // @ts-ignore importScripts available only when remote debugging
+    'function' === typeof importScripts &&
+    // @ts-ignore WorkerNavigator available only when remote debugging
+    navigator instanceof WorkerNavigator
+  );
+}
```

Comment:
Maybe let's do it the same way as in Reanimated? See [`isChromeDebugger`](https://github.com/software-mansion/react-native-reanimated/blob/688a244b3826b0f6cde52e9490eee5fe0d3e9bf9/src/reanimated2/PlatformChecker.ts#L7-L9)

Reply:
Done in [b51acf5](https://github.com/software-mansion/react-native-gesture-handler/pull/1929/commits/b51acf592ae4669ecacff879207c0a5a94b865e9).